back/dora/structures/csv_import.py

```diff
@@ -240,38 +170,41 @@ def invite_users(self, structure, emails):
                         is_admin=True,
                     )
 
-                    self.stdout.write(f"{email} invité·e comme administrateur·rice")
+                    print(f"{email} invité·e comme administrateur·rice")
                     send_invitation_email(
                         member,
                         "L’équipe DORA",
                     )
 
-    def add_labels(self, structure, labels):
+    def add_labels(
+        self, structure: Structure, labels: List[StructureNationalLabel]
+    ) -> None:
         for label in labels:
             if label not in structure.national_labels.all():
-                self.stdout.write(f"Ajout du label {label.value}")
+                print(f"Ajout du label {label.value}")
                 structure.national_labels.add(label)
 
-    def create_services(self, structure, models):
+    def create_services(
+        self, structure: Structure, models: List[ServiceModel], importing_user: User
+    ) -> None:
         for model in models:
             if not structure.services.filter(model=model).exists():
                 service = instantiate_service_from_model(
-                    model, structure, self.bot_user
-                )
-                self.stdout.write(
-                    f"Ajout du service {service.name} ({service.get_frontend_url()})"
+                    model, structure, importing_user
                 )
+                print(f"Ajout du service {service.name} ({service.get_frontend_url()})")
+                self.created_services_count += 1
 
-    def _get_or_create_branch(self, name, siret, parent_structure, **kwargs):
+    def _get_or_create_branch(
+        self, name: str, siret: str, parent_structure: Structure, **kwargs
+    ) -> Structure:
         try:
             if siret:
                 branch = Structure.objects.get(siret=siret)
             else:
                 branch = Structure.objects.get(parent=parent_structure, name=name)
 
-            self.stdout.write(
-                f"La branche {branch.name} ({branch.get_frontend_url()}) existe déjà"
-            )
+            print(f"La branche {branch.name} ({branch.get_frontend_url()}) existe déjà")
```

Comment:
```suggestion
            print(f"L’antenne {branch.name} ({branch.get_frontend_url()}) existe déjà")
```

Terme anglais : branch
Terme français : antenne